pandas/core/frame.py

```diff
@@ -7590,8 +7588,8 @@ def _get_data(axis_matters):
                     data = self._get_bool_data()
             else:  # pragma: no cover
                 msg = (
-                    "Generating numeric_only data with filter_type {f}"
-                    "not supported.".format(f=filter_type)
+                    f"Generating numeric_only data with filter_type {filter_type}"
+                    "not supported."
```

Comment:
this appears to have a missing space in the message. do we have a test for this message?